.github/workflows/build_and_push_production.yml

```diff
@@ -41,6 +42,13 @@ jobs:
         run: |
           docker push $IMAGE_NAME
 
+      - name: Deploy new image
+        run: |
+          aws lambda update-function-code \
```

Comment:
To make it easier to rollback changes, each time you build the image, could you tag and push it with the git commit SHA, and then deploy that specific tag.  

This way if something breaks, we can just rollback to the last commit SHA while we investigate.

Reply:
Oh interesting that's convenient!